components/Navigation/index.js

```diff
@@ -1,27 +1,38 @@
 import Link from "next/link";
 import styled from "styled-components";
 import HomeSvg from "@/public/home.svg";
+import PlusSvg from "@/public/plus.svg";
+import { useRouter } from "next/router";
 
 export default function Navigation() {
+  const router = useRouter();
   return (
     <StyledNavigation>
       <StyledList>
         <li>
           <StyledLink href="/">
-            <StyledSvg />
+            <StyledHomeSvg isActive={router.pathname === "/"} />
```

Comment:
When you pass a prop to a styled-component, you should prefix it with "$", otherwise, your console will show an error:

`Warning: React does not recognize the `isActive` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `isactive` instead. If you accidentally passed it from a parent component, remove it from the DOM element.`

```suggestion
            <StyledHomeSvg $isActive={router.pathname === "/"} />
```

Reply:
thanks for the advice - we add this to our code